File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        tb={}
        for idx,num in enumerate(nums):
            if target-num in tb:
                return [idx,tb[target-num]]
            tb[num]=idx
        

File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    
    var ans []int;
    
    for i, num := range nums{
        for j,num_ := range nums{
            if i != j && num + num_ == target{
                ans = append(ans,i)
            }
        }
    }
    return ans
    
}